app/Http/Controllers/PaymentController.php

```diff
@@ -85,6 +90,18 @@ public function paymentConfirmation(Request $request){
     }
 
     public function proceed(Request $request){
+      $order=Order::create([
```

Comment:
This function was being used after login/sign up.

Please use another method/controller instead. :)

Reply:
used new controller.